src/components/Profile/Profile.container.tsx

```diff
@@ -1,36 +1,167 @@
-import { FC, SyntheticEvent, useState } from "react";
+import { FC, SyntheticEvent, useEffect, useState } from "react";
 import { TabType } from "../../models/ui.model";
 import Profile from "./Profile";
 import GridOnIcon from "@mui/icons-material/GridOn";
 import BookmarkIcon from "@mui/icons-material/Bookmark";
 import PlayCircleOutlineIcon from "@mui/icons-material/PlayCircleOutline";
+import useAuth from "../../hooks/useAuth";
```

Comment:
Read about path mapping https://medium.com/@danil_chepelev/how-to-setup-path-aliases-with-react-typescript-and-webpack-54e579c189e1